src/components/_global/BalProgressBar/BalProgressBar.vue

```diff
@@ -23,27 +24,40 @@ const props = withDefaults(defineProps<Props>(), {
  */
 const barClasses = computed(() => ({
   [`h-${props.size}`]: true,
-  [`bg-${props.color}-400`]: true
+  [`bg-${props.color}-500 dark:bg-${props.color}-400`]: true
```

Comment:
I realise Pon's designs might have said something different, but for now we should probably stick to green-400 since that's what we seem to be using as default everywhere else. Not sure why that is, longer term we should update the green color scale and use 500 as the default.
<img width="830" alt="Screenshot 2021-11-12 at 10 45 16" src="https://user-images.githubusercontent.com/2406506/141454882-374f381c-83cc-4409-bee5-67f3117123ac.png">

Reply:
Makes sense yeah. What about the yellow tho? stick with 500 as well without changes to the dark one?

Reply:
Because we're not really using the yellow much elsewhere, if 500 looks good lets just stick with that.